demos/projects/ESPRESSIF/aziotkit/config/demo_config.h

```diff
@@ -127,6 +127,12 @@
  *  - DigiCert Global Root G2 
  *  - Microsoft RSA Root Certificate Authority 2017
  *
+ * @warning Hard coding certificates is not recommended by Microsoft as a best
+ * practice for production scenarios. Please see our document here for notes on best practices.
+ * https://github.com/Azure-Samples/iot-middleware-freertos-samples/blob/main/docs/certificate-notice.md
+ *
+ * These certs are DER formats of the above listed certificates.
```

Comment:
Not sure if we should include an explicit call to an openssl function here. What about just describing the need for DER and documenting how to get it in the certificate-notice.md ? Or somewhere else other than the source code/headers.

Same question for other places below.

Reply:
That mainly came from Chris's comment here:
https://github.com/Azure-Samples/iot-middleware-freertos-samples/pull/139#issuecomment-1026381756

I don't mind if they are or aren't here.

Reply:
Maybe we can add all certificate documentation in the MD file simplifying the comment to the warning only?